src/core/analyzer.ts

```diff
@@ -77,6 +78,17 @@ export class Analyzer {
     }
   }
 
+  /**
+   * Returns the best analyzer for the given directory.
+   *
+   * Currently this just creates a simple analyzer with a FS loader rooted
+   * at the given directory, but in the future it may take configuration from
+   * files including polymer.json or similar.
```

Comment:
Hm, IMO if you want maximum stability you just shouldn't use this method. This method is definitely closer to the Do What I Mean direction, but not in ways that seem like they'd be problematic. Hm, I guess when we add support for including and excluding files from analysis that will be an issue.

I'm not sure if adding a flag is going to be helpful though, because this is specifically for those cases where we _don't_ have any good way for the user of the final tool to configure how they want the analyzer to be constructed except through their config files.